lib/slack.js

```diff
@@ -58,10 +58,10 @@ export default class SlackData extends EventEmitter {
       return this.retry();
     }
 
-    let total = users.length;
+    let total = users.length - 1;
```

Comment:
maybe, but this is much simpler and actually produces the same result :sunglasses: